data/sailfish/hammerhead/resolutions.ini

```diff
@@ -0,0 +1,439 @@
+[image-high-16:9]
+capture = 1920x1080
+viewfinder=1280x720
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+[image-medium-16:9]
+capture = 1280x720
+viewfinder=1280x720
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+[image-medium-16:10]
+capture = 1280x768
+viewfinder=1280x768
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+[image-low-16:10]
+capture = 800x480
+viewfinder=800x480
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+[image-high-4:3]
+capture = 3264x2448
+viewfinder=1280x960
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+#[image-high1-4:3]
+#capture = 3200x2400
+#viewfinder=1280x960
+#preview=960x720
+#fps=15
+#night = -1
+#device=0
+#mode=image
+
+#[image-high2-4:3]
+#capture = 2592x1944
+#viewfinder=1280x960
+#preview=960x720
+#fps=15
+#night = -1
+#device=0
+#mode=image
+
+[image-high3-4:3]
+capture = 2048x1536
+viewfinder=1280x960
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+[image-medium-4:3]
+capture = 1600x1200
+viewfinder=1280x960
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+#[image-medium1-4:3]
+#capture = 1280x960
+#viewfinder=1280x960
+#preview=960x720
+#fps=15
+#night = -1
+#device=0
+#mode=image
+
+[image-low-4:3]
+capture = 1024x768
+viewfinder=1024x768
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+[image-low1-4:3]
+capture = 800x600 
+viewfinder=800x600
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+#[image-micro-4:3]
+#capture = 640x480 
+#viewfinder=640x480
+#preview=960x720
+#fps=15
+#night = -1
+#device=0
+#mode=image
+
+#[image-micro1-4:3]
+#capture = 320x240
+#viewfinder=320x240
+#preview=960x720
+#fps=15
+#night = -1
+#device=0
+#mode=image
+
+[image-low-3:2]
```

Comment:
I see two duplicate resolutions under this aspect ratio. If i comment this block - both disappear